specification/app/resource-manager/Microsoft.App/preview/2022-06-01-preview/CommonDefinitions.json

```diff
@@ -903,6 +903,69 @@
           "description": "Application-specific metadata in the form of key-value pairs."
         }
       }
+    },
+    "BillingMeter": {
```

Comment:
`BillingMeters` should be in int's own file. Make sure you use the `x-ms-examples` attribute to reference the example file `BillingMeters_Get.json`

Reply:
Fixed.

Reply:
Fixed